### PR TITLE
testing/libfixposix: Correct Packaging

### PR DESCRIPTION
**Pull request by durrendal**
Add a symlink of libfixposix.3.2.0 -> libfixposix.so

### CODE REVIEW

File: testing/libfixposix/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Contributor: Will Sinatra <wpsinatra@gmail.com>
 pkgname=libfixposix
 pkgver=0.4.3
-pkgrel=0
+pkgrel=1.0
```
Comment (jjsearle) [inline testing/libfixposix/APKBUILD]: Why is the pkgrel `1.0` instead of `1`? I don't think this is valid.

Comment (durrendal) [reply testing/libfixposix/APKBUILD]: I honestly couldn't remember if it was 1.0 or 1, but that's an easy fix at least.

Comment (durrendal) [reply testing/libfixposix/APKBUILD]: Regardless, it's set to 1 instead of 1.0 now.

**Comment by Cogitri** [comment]
Honestly not quite sure what you're trying to do here, why would you want a symlink like that in your `/`? It provides a symlink in `/usr/lib` as it should AFAICS: https://pkgs.alpinelinux.org/contents?branch=edge&name=libfixposix&arch=x86_64&repo=testing

**Comment by durrendal** [comment]
The purpose was actually to create a symlink in /usr/lib/, certain programs look for libfixposix.so specifically, rather than the .3 or .3.2.0 version.

**Comment by durrendal** [comment]
Verified I missed something, .so is provided by libfixposix-dev and I just wasn't well enough aware.

**Pull request closed without merging**